FEC-10909 FEC-12435 Ads progress indicator/seekbar fixes

chausov opened the pull request:
Solves:
FEC-10909 progress indication added for ads
FEC-12435 seeking disabled while ads are playing

chausov committed:
FEC-10909 progress indication added for ads
FEC-12435 seeking disabled while ads are playing

x-NR-x commented on Sources/UI/KPMediaPlayer.swift:
```diff
@@ -378,6 +379,32 @@ extension KPMediaPlayer {
         }
     }
     
+    private func handleAdEvents() {
```
Why was this separated? The adStarted and adComplete events are already in the registerAdEvents function.
And please use the type alias KPAdEvent and not the Playkit AdEvent.

chausov committed:
fixes

chausov committed:
Merge branch 'develop'

x-NR-x commented on Sources/UI/KPMediaPlayer.swift:
```diff
@@ -410,8 +410,10 @@ extension KPMediaPlayer {
+                    self.mediaProgressSlider.maximumTrackTintColor = UIColor.orange
                 case is KPAdEvent.AdComplete:
                     self.mediaProgressSlider.isEnabled = true
+                    self.mediaProgressSlider.maximumTrackTintColor = UIColor.lightGray.withAlphaComponent(0.5)
```
I think this needs to be in ad skipped as well.

chausov committed:
AdSkipped handeling for seekbak color

chausov committed:
Ads current time and duration labels fixes

x-NR-x reviewed (changes requested):
Please fix:
- The duration label is not updated back to the media value after the ad (pre/mid).
- The 'mediaProgressSlider.bufferValue' should not be shown upon an ad, but should be seen after the ad.

chausov committed:
- The duration label is not updated back to the media value after the ad (pre/mid).
- The 'mediaProgressSlider.bufferValue' should not be shown upon an ad, but should be seen after the ad.

chausov committed:
Merge branch 'develop'

chausov commented:
> Please fix:
> 
> * The duration label is not updated back to the media value after the ad (pre/mid).
> * The 'mediaProgressSlider.bufferValue' should not be shown upon an ad, but should be seen after the ad.

Fixed
